File: 2020/day02a.py
def main():
    lines = open('input.txt','r').read().splitlines()
    correct = 0
    for line in lines:
        min_count = int(line[:line.index('-')])
        max_count = int(line[line.index('-') + 1 : line.index(' ')])
        target = line[line.index(' ') + 1]
        if min_count <= line[line.index(':') + 2 :].count(target) <= max_count:
            correct += 1
    print(correct)
    
if __name__ == '__main__':
    main()

File: 2022/day02a.jl
function main()
    lines = readlines(open("input.txt"))
    score = 0

    for line in lines
        my_move = codepoint(line[3]) - 88
        opponent_move = codepoint(line[1]) - 65

        score += my_move + 1
        score += 3 * mod(my_move - opponent_move + 1, 3)
    end
    
    println(score)
end

main()
